test/tests/api/redfish_1_0/event_service_tests.py

```diff
@@ -86,19 +86,28 @@ def test_get_event_subscription(self):
     def test_submit_test_event(self):
         """ Testing POST /EventService/SubmitTestEvent  """
         global task
-        server = Httpd(port=int(HTTPD_PORT), handler_class=self.EventServiceHandler)
+        # Suppose rackhd and test stack have the same localhost
+        server = Httpd(port=int(self.__httpd_port), handler_class=self.EventServiceHandler)
         task = WorkerThread(server,'httpd')
         worker = WorkerTasks(tasks=[task], func=self.__httpd_start)
         worker.run()
-        
-        # forward port for services running on a guest host
-        session = open_ssh_forward(self.__httpd_port) 
-        
         redfish().test_event(body={})
         worker.wait_for_completion(timeout_sec=60)
-        session.logout()
+        if task.timeout:
+            # Else port forward rackhd -> localhost
+            server = Httpd(port=int(HTTPD_PORT), handler_class=self.EventServiceHandler)
```

Comment:
the worker variable get 2 worker thread. When get the second one, the first thread will lost reference. consider use another variable name or do recycle on thread 1.